src/traces/sankey/attributes.js

```diff
@@ -168,6 +168,14 @@ var attrs = module.exports = overrideAll({
     },
 
     link: {
+        arrowlen: {
```

Comment:
Start arrows may be another effect that one wants to add in future.
@alexcjohnson what do think if we rename this attribute to be something like `endarrowlen` or `endlen`?

Reply:
What would that look like? Are there examples of this created elsewhere? I'd rather not complicate the naming like that unless we have a clear plan to implement it. Also even if we did, I imagine we could have `arrowlen` be the default that gets inherited by both `startarrowlen` (if it was enabled) and `endarrowlen`.